### PR TITLE
docs: add docs content for LSP8

### PR DESCRIPTION
**Pull request by CJ42**
# What does this PR introduce?

added more documentation for LSP8 - Identifiable Digital Asset

- current problems and limitations around ERC721
- figures for different types of `tokenId` (number, bytes32 and ERC725Y contract)
- added figures for ERC721 metadata security
- add explanations of solutions solved by LSP8 + its new features

### CODE REVIEW

File: docs/standards/nft-2.0/03-LSP8-Identifiable-Digital-Asset.md [hunk 1]
```diff
@@ -4,37 +4,92 @@ sidebar_label: 'LSP8 - Identifiable Digital Asset'
...
 
-One of the key difference of LSP8 is that each newly created `tokenId` (NFT) has its own metadata, that describes its uniqueness. Such metadata can be a reference either:
+LSP8 assets are similar to [ERC721](https://eips.ethereum.org/EIPS/eip-721) tokens (NFTs). What makes them different is their underlying base, making them more customisable and their metadata more upgradable over time. This is what makes them a new generation of unique assets known as **NFT 2.0**.
 
-- _off-chain_ (a **JSON file url** stored on IPFS, a public / private registry, etc...)
-- _on-chain_ (an other **ERC725Y smart contract**)
+### NFT Metadata
 
-It is possible to know where the metadata for a specific tokenID is located by simply querying the LSP8 contract, giving the `tokenID` as an argument.
+One of the key differences of LSP8 is that it allows representing each NFT as more than just a unique `tokenID` number. Each newly created `tokenId` (NFT) has its own metadata, that describes its uniqueness. Such metadata can be a reference either:
```
Comment (Hugoo) [inline docs/standards/nft-2.0/03-LSP8-Identifiable-Digital-Asset.md:55]: do you want to write `tokenId` or `tokenID` ? I saw both in this PR - maybe you want to always use one or another

File: docs/standards/nft-2.0/03-LSP8-Identifiable-Digital-Asset.md [hunk 2]
```diff
@@ -4,37 +4,92 @@ sidebar_label: 'LSP8 - Identifiable Digital Asset'
...
+
+:::
+
+An LSP8 NFT can be represented in multiple ways, depending on the type of `tokenId` used.
+
+- a **number** (`uint256`), that increments on each newly minted NFT.
+- a unique `bytes32` value (representing a _serial number_ for instance).
+- an ERC725Y contract.
+
+![](../../../static/img/lsp8-tokenid-number.jpeg)
```
Comment (Hugoo) [inline docs/standards/nft-2.0/03-LSP8-Identifiable-Digital-Asset.md:78]:
```suggestion
![put alt text here](../../../static/img/lsp8-tokenid-number.jpeg)
```

File: docs/standards/nft-2.0/03-LSP8-Identifiable-Digital-Asset.md [hunk 3]
```diff
@@ -4,37 +4,92 @@ sidebar_label: 'LSP8 - Identifiable Digital Asset'
...
+
+---
+
+![](../../../static/img/lsp8-tokenid-serial-number.jpeg)
```
Comment (Hugoo) [inline docs/standards/nft-2.0/03-LSP8-Identifiable-Digital-Asset.md:82]:
```suggestion
![put alt text here](../../../static/img/lsp8-tokenid-serial-number.jpeg)
```

File: docs/standards/nft-2.0/03-LSP8-Identifiable-Digital-Asset.md [hunk 4]
```diff
@@ -4,37 +4,92 @@ sidebar_label: 'LSP8 - Identifiable Digital Asset'
...
+
+When it comes to Non Fungible Tokens (NFTs), one of the key characteristics that make such assets uniques is their **metadata.**
+
+In their current state, such assets are created as ERC721 tokens. However, they present certain limitations:
+
+- _the Metadata attached to the NFT (represented with `tokenURI`) is a **reference** to a file._
+
+This raises several questions around the location of where the token metadata is stored. Is a central service used to store the metadata? What about if such service goes down?
+
+![](../../../static/img/erc721-metadata-security.png)
```
Comment (Hugoo) [inline docs/standards/nft-2.0/03-LSP8-Identifiable-Digital-Asset.md:23]:
```suggestion
![put alt text here](../../../static/img/erc721-metadata-security.png)
```

File: docs/standards/nft-2.0/03-LSP8-Identifiable-Digital-Asset.md [hunk 5]
```diff
@@ -4,37 +4,92 @@ sidebar_label: 'LSP8 - Identifiable Digital Asset'
...
+![](../../../static/img/lsp8-tokenid-serial-number.jpeg)
+
+### NFTs as ERC725Y contracts
+
+If each NFT is represented by its own ERC725Y contract (= metadata contract), this contract contains informations like:
 
 - the `address` of the contract that minted this NFT.
 - the `tokenId` of this NFT.
 
+![](../../../static/img/lsp8-tokenid-erc725y.jpeg)
```
Comment (Hugoo) [inline docs/standards/nft-2.0/03-LSP8-Identifiable-Digital-Asset.md:91]: missing alt

**Review by Hugoo — APPROVED** [review]
don't forget to always set alt for images

**Review by YamenMerhi — APPROVED** [review]

**Pull request merged**